sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -2609,6 +2610,7 @@ public CaseReferenceDto convertToReferenceDto(Case source) {
 		return dto;
 	}
 
+	@RolesAllowed(UserRight._EXTERNAL_VISITS)
```

Comment:
This only allows `UserRight._EXTERNAL_VISITS` to use this method.

Reply:
It should, but it doesn't seem to. I am about to add a comment to the PR listing some of my own open questions. Maybe we can then use this as a basis for discussion.